main.c

```diff
@@ -16,194 +19,265 @@
  *
  *	- parses through initial command line
  *	- in the absence of a file to parse, open main page, which can either do one of 4 things:
- *		- load a new dictionary (will replace the one that is parsed, if
+ *		- load a new dictionary (will replace the one that is parsed)
  *		- quit
- *		- 
- *
- *	- launch either interactive or batch mode
- *	- save file
+ *		- launch either interactive or batch mode
+ *		- save file
  *
  *  Sample inputs:
  *
  *  One input
  *	> ./spellcheck: opens main page
  *
  *  Two inputs
- *	> Not possible to determine whether entered filename is for dict or editing
- *	> Print error messages and exit
-
- *  Three inputs
- *	> ./spellcheck -d [dictname.txt]: stores 
+ *	> ./spellcheck [filename.txt]
+ *
+ *	Other flags:
+ *	> -d [dictname.txt]: changes dictionary
+ *
+ *	> -i/-q : interactive, quiet mode
  *
- *	> ./spellcheck -i [filename.txt]: interactive mode
+ *  > --show-suggestions: verbose mode
  *
- *	> ./spellcheck -q [filename.txt]: quiet mode
+ *	> -s [filename.txt]: file saving destination 
  *
- *	> ./spellcheck -v [filename.txt]: verbose mode
+ *	> -c : colored strings
+ *
+ *  > -v : logging
  *
  *  ...
  *  and so on, with different combinations.
  *
- *  Largest possible number of argc: 7
- *	> ./spellcheck -d my_dict.txt -q misspelled.txt -s savefilename.txt
- *
+ *  Largest possible number of argc: 9
+ *	> ./spellcheck file.txt -d my_dict.txt -q misspelled.txt -s savefilename.txt -c
  */
 
-char* modename(int mode)
-{
-	switch (mode) {
+char *modename(int *mode) {
```

Comment:
What is this function used for?

Reply:
it had been replaced with a shellstrings function. removed.